themes/bootstrap3/js/record.js

```diff
@@ -298,6 +298,74 @@ function removeCheckRouteParam() {
   }
 }
 
+function initTruncate(_holder, _target, _fill = function(m) { return m } ) {
+  var holder = typeof _holder === 'undefined' ? $(document) : $(_holder);
+  var target = $(_target);
+  var targetElemName = target.length && target.prop('tagName').toLowerCase();
+  var targetClass = target.length ? ` ${target.prop('class')}` : '';
+  var rowCount = holder.data('rows') || 3;
+  var moreLabel = holder.data('more-label') ? holder.data('more-label') : VuFind.translate('show_more');
+  var lessLabel = holder.data('less-label') ? holder.data('less-label') : VuFind.translate('show_less');
+  var btnSize = holder.data('btn-size') ? ` ${holder.data('btn-size')}` : '';
+  var topToggle = holder.data('top-toggle') || Infinity;
+  var inPlaceToggle = holder.data('in-place-toggle') || false;
+
+  holder.each(function parseHolder() {
+    var self = $(this);
+    var numRows = self.find(target).length || 0;
+    var hasTopToggle = numRows > topToggle;
+    // Truncate only if there's more than one line to hide
+    var shouldTruncate = rowCount < numRows || false;
+    self.find(target).each(function hideRows(i) {
+      var truncateAfter = rowCount - 1;
+      if (i == truncateAfter) {
+        $(this).addClass('truncate-after');
+      }
+      if(i > truncateAfter && shouldTruncate) {
+        $(this).hide();
+        $(this).addClass('truncate-toggle');
+      }
+    });
+
+    if (shouldTruncate) {
+      var btn = '<button type="button" class="more-link btn' + btnSize + '">' + moreLabel + ' <i class="fa fa-arrow-down" aria-hidden="true"></i></button><button type="button" class="less-link btn' + btnSize + '">' + lessLabel + ' <i class="fa fa-arrow-up" aria-hidden="true"></i></button>';
+      var btnLessTop = '<button type="button" class="less-link-top btn' + btnSize + '">' + lessLabel + ' <i class="fa fa-arrow-up" aria-hidden="true"></i></button>';
+      var btnWrapper = $('<' + targetElemName + ' class="more-less-btn-wrapper' + targetClass + '"></' + targetElemName + '>');
```

Comment:
I don't have any issues with this section but the community should have a discussion on how to handle sub element generation in the future. A few options include pulling in PHP templates, [HTML5 templates](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_templates_and_slots), or custom web components.

Reply:
I think something needs to be done with the backticks since eslint doesn't let them through. Either change the rules or avoid backticks?